src/Main.java

```diff
@@ -1,5 +1,89 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
-    }
-}
\ No newline at end of file
+
+        System.out.println("Задание 1");
+        int ageForExample = 27;
+        if (ageForExample >= 18) {
+            System.out.println("Если возраст человека равен " + ageForExample + " , то он совершеннолетний");
+        } else {
+            System.out.println("Если возраст человека равен " + ageForExample + " , то он не достиг совершеннолетия, нужно немного подождать");
+        }
+
+        System.out.println("\nЗадание 2");
+        int temperatureForExample = 8;
+        if (temperatureForExample <= 5) {
+            System.out.println("На улице " + temperatureForExample + " градусов, холодно, нужно надеть шапку");
+        }
+
+        if (temperatureForExample > 5) {
+            System.out.println("На улице " + temperatureForExample + " градусов, сегодня тепло можно идти без шапки");
+        }
+
+        System.out.println("\nЗадание 3");
+        int speedForExample = 45;
+        if (speedForExample >= 60) {
+            System.out.println("Если скорость " + speedForExample + " км/ч, то придется заплатить штраф");
+        } else {
+            System.out.println("Если скорость " + speedForExample + " км/ч, то можно ездить спокойно");
+        }
+
+        System.out.println("\nЗадание 4");
+        int ageForEducationExample = 27;
```

Comment:
можно было не заводить новую переменную для возраста, а просто использовать уже созданную ранее
```suggestion
        ageForExample = 27;
```